Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.xaml

```diff
@@ -160,6 +161,7 @@
                                     Content="&#xE73F;"
                                     FontFamily="Segoe MDL2 Assets"
                                     FontSize="12" 
+                                    AutomationProperties.Name="EnlargeButton"
```

Comment:
We should add x:Uid for both buttons here.  Also AutomationProperties.Name should never have the type repeated in them as the Narrator will announce `<text> button` already, so this would cause duplication of 'button'.

Reply:
@michael-hawker  You are right, but as other classes as BladeItem use this name as a reference all the callings to this button will have to be changed. I'll try to have this changes done by tomorrow. We just submitted some commits addressing most of the changes requested here. could you check them?